app/models/user.rb

```diff
@@ -1,9 +1,13 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] As requested by the previous reviewer, when creating an account from the signup form, an error is encountered which makes the sign-up unsuccessful. So make sure that you fix this error as seen in the screenshot below. :point_down: 
  
  ![image](https://user-images.githubusercontent.com/92303891/233803923-76240e3f-0cbf-4e6d-8912-ed0fa82d0086.png)

- [ ] When creating an account from the signup form, make sure that a user is able to confirm the email used for the sign-up.

Reply:
All issues have been corrected.